tools/antithesis/test_format.dockerfile

```diff
@@ -5,7 +5,7 @@ RUN mkdir -p bin/test/format
 RUN mkdir -p /data/RUNDIR
 COPY cmake_build/test/format/t bin/test/format/
 COPY tools/voidstar/lib/libvoidstar.so tools/voidstar/lib/
-COPY cmake_build/libwiredtiger.so.11.2.0 bin/
+COPY cmake_build/libwiredtiger.so.* bin/
```

Comment:
This will also copy across our gdb auto-load script `libwiredtiger.so.11.3.0-gdb.py`.
It's not a big issue, worst case gdb will complain about missing imports when we open an antithesis core in gdb, but it'd be nice to clean this up ahead of time.
I think deletion of `libwiredtiger.so.*-gdb.py` is the easiest approach.

Reply:
Please see 69a5506, I went for a stricter match when copying.